app/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -979,10 +981,14 @@ public void onItemClicked(OCFile file) {
                         } else {
                             Log_OC.d(TAG, "no public key for " + user.getAccountName());
 
-                            SetupEncryptionDialogFragment dialog = SetupEncryptionDialogFragment.newInstance(user,
-                                                                                                             position);
-                            dialog.setTargetFragment(this, SetupEncryptionDialogFragment.SETUP_ENCRYPTION_REQUEST_CODE);
-                            dialog.show(getFragmentManager(), SetupEncryptionDialogFragment.SETUP_ENCRYPTION_DIALOG_TAG);
+                            FragmentManager fragmentManager = getFragmentManager();
```

Comment:
```suggestion
                            FragmentManager fragmentManager = getParentFragmentManager();
```
`getFragmentManager()` is deprecated, let's remove it while we're changing this